setup.py

```diff
@@ -83,7 +83,7 @@ def readme():
         # Until this issue is closed
         # https://github.com/googleapis/google-cloud-python/issues/10566
         "google-auth>=1.25.0,<3.0.0dev",  # Work around pip wack.
-        "google-cloud-bigquery>=2.24.1",
+        "google-cloud-bigquery>=2.25.2,<3.0.0dev",
```

Comment:
Presumably for https://github.com/googleapis/python-bigquery/pull/930 ?

I'm okay with this, but would like to consider the alternatives.

Is there logic that we'd have to change to support older versions of `google-cloud-bigquery`? Or can we disable a few failing tests when using Python 3.6 (which constrains to the older package)?

Reply:
> Is there logic that we'd have to change to support older versions of `google-cloud-bigquery`?

We could monkey-patch the 2 broken functions.

> Or can we disable a few failing tests when using Python 3.6 (which constrains to the older package)?

We could, but then folks might be surprised when valid structs weren't handled correctly.

Reply:
OK, let's leave this as you've done it. I like the stricter requirements to either of these options.